assets/css/styles.css

```diff
@@ -343,3 +515,6 @@
     top: 0px;
   }
 }
+
+

```

Comment:
Can you remove these unwanted lines @Neutx 
```suggestion
```

Reply:
Line has been removed